x/ecocredit/marketplace/keeper/features/msg_buy_direct.feature

```diff
@@ -376,9 +380,54 @@ Feature: Msg/BuyDirect
     Scenario: EventBuyDirect is emitted
       Given alice created a sell order with id "1"
       When bob attempts to buy credits with sell order id "1"
-      Then expect event buy direct with properties
+      Then expect no error
+      And expect event buy direct with properties
       """
       {
         "sell_order_id": 1
       }
       """
+
+  Rule: buyer fees are deducted from buyer, seller fees are deducted from seller, and both go to fee pool or get burned
+    Background:
+      Given a credit type
+
+    Scenario: fees go to fee pool
+      Given alice created a sell order with quantity "10" and ask price "10foo"
+      * bob has bank balance "110foo"
+      * alice has bank balance "0foo"
+      * buyer fees are 0.1 and seller fees are 0.05
+      When bob attempts to buy credits with quantity "10" and bid price "10foo"
+      Then expect no error
+      * expect alice bank balance "95foo"
+      * expect bob bank balance "0foo"
+      * expect fee pool balance "15foo"
+
+    Scenario: fees get burned
```

Comment:
We can emit the burn event here. Don't have a strong preference.

Reply:
It seems like a burn event should be emitted but it just isn't clear to me if that is already happening. Does the bank keeper do that for us? If so, then maybe we don't need to include in tests.

Reply:
The bank keeper does include burn events, but since `MsgBuyDirect` now also includes the fees paid and we can assume that if the denom is `uregen` that it was a burn, is it okay if we just leave it at that and not add another event?

Reply:
What are the pros and cons of adding a specific event here? seems like for users, it is useful to have a specific event.

Reply:
I don't know if there's really a pro to adding it. The con is that it just adds duplicitous events which takes up extra storage. The bank keeper and marketplace module already emit events containing the relevant data, so I think we should leave it as is.

Reply:
Sorry I might have created some confusion here. I just wanted to make sure that a burn event does exist. I agree we do not need a duplicate event. This would all be very clear if the test scenario included checking that the burn event is emitted but no worries if this is non-trivial to add.